File: clients/dotnet/enums/ResolutionQuality.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace Avalara.AvaTax.RestClient
{
    /// <summary>
    /// 
    /// </summary>
    public enum ResolutionQuality
    {
        /// <summary>
        /// 
        /// </summary>
        NotCoded,

        /// <summary>
        /// 
        /// </summary>
        External,

        /// <summary>
        /// 
        /// </summary>
        CountryCentroid,

        /// <summary>
        /// 
        /// </summary>
        RegionCentroid,

        /// <summary>
        /// 
        /// </summary>
        PartialCentroid,

        /// <summary>
        /// 
        /// </summary>
        PostalCentroidGood,

        /// <summary>
        /// 
        /// </summary>
        PostalCentroidBetter,

        /// <summary>
        /// 
        /// </summary>
        PostalCentroidBest,

        /// <summary>
        /// 
        /// </summary>
        Intersection,

        /// <summary>
        /// 
        /// </summary>
        Interpolated,

        /// <summary>
        /// 
        /// </summary>
        Rooftop,

        /// <summary>
        /// 
        /// </summary>
        Constant,


    }
}

File: clients/php/enums/ResolutionQuality.php
<?php
class ResolutionQuality extends AvaTaxEnum 
{

    const NotCoded = "NotCoded";
    const External = "External";
    const CountryCentroid = "CountryCentroid";
    const RegionCentroid = "RegionCentroid";
    const PartialCentroid = "PartialCentroid";
    const PostalCentroidGood = "PostalCentroidGood";
    const PostalCentroidBetter = "PostalCentroidBetter";
    const PostalCentroidBest = "PostalCentroidBest";
    const Intersection = "Intersection";
    const Interpolated = "Interpolated";
    const Rooftop = "Rooftop";
    const Constant = "Constant";
}
?>
